openmc/cell.py

```diff
@@ -528,11 +529,12 @@ def clone(self, memo=None):
         # If no nemoize'd clone exists, instantiate one
         if self not in memo:
             # Temporarily remove paths
-            paths = self.paths
+            paths = self._paths
             self._paths = None
```

Comment:
In the constructor, ``self._paths`` is set to ``[]``, but here it is set to ``None``.  It doesn't change the exception handling, but it would be nice if they were consistent.